packages/terra-list/tests/jest/__snapshots__/List.test.jsx.snap

```diff
@@ -1,212 +1,43 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`correctly applies the theme context className 1`] = `
-<ThemeContextProvider
-  theme={
-    Object {
-      "className": "orion-fusion-theme",
-    }
-  }
->
-  <List
-    dividerStyle="none"
-    paddingStyle="none"
-    role="none"
-  >
-    <ul
-      className="list orion-fusion-theme"
-    />
-  </List>
-</ThemeContextProvider>
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
```

Comment:
we sure this replacement on the jest test is right?

Reply:
Oops, this was updated accidentally, fixed [9e651a0](https://github.com/cerner/terra-core/pull/3703/commits/9e651a0db2c56fd4025d8c79baf20d07dfc91e24)